framework/src/controller/application.js

```diff
@@ -167,6 +169,8 @@ class Application {
 
 		this.registerModule(ChainModule, {
 			registeredTransactions: this.getTransactions(),
+			lastCommitId: this.config.app.lastCommitId,
+			buildVersion: this.config.app.buildVersion,
```

Comment:
Both of these values are used in the `http_api` module as well, so don't pass it only to chain module.

See in the function `_compileAndValidateConfigurations` and you will find an object `appConfigToShareWithModules`, please add these values to it, so it would be shared by all modules. 

Then make sure the `http_api` module must use these values in the API.